src/mmrelay/migrate.py

```
@@ -0,0 +1,1893 @@
+        "action": "move",
+    }
+    if errors:
+        result["error"] = "; ".join(errors)
+    return result
+
+
+def migrate_gpxtracker(
+    legacy_roots: list[Path],
+    new_home: Path,
+    dry_run: bool = False,
+    force: bool = False,
```

Ruff (0.14.14)

[warning] Unused function argument: `force`

(ARG001)